typescript/src/client/evm/evm.ts

```diff
@@ -258,12 +259,15 @@ export class EvmClient implements EvmClientInterface {
    *          ```
    */
   async createSmartAccount(options: CreateSmartAccountOptions): Promise<SmartAccount> {
-    const openApiSmartAccount = await CdpOpenApiClient.createEvmSmartAccount(
-      {
-        owners: [options.owner.address],
-      },
-      options.idempotencyKey,
-    );
+    const openApiSmartAccount = await (() => {
```

Comment:
we don't need the IIFE here. it can just be like the left hand side, but with the `name: options.name` included in the params